src/m_menu.c

```diff
@@ -3916,6 +3917,10 @@ static void M_SetMidiPlayer(void)
   S_RestartMusic();
 }
 
+static const char *uncapped_strings[] = {
```

Comment:
Hm, how do we handle this in Crispy? I mean, if the min/max range for the `fpslimit` value is the limiting factor here, we could just call a sanitizer function after that value has been set, which allows either `0` for "disable" or any other value higher than `35`.

Reply:
We coerce values from to 1 to 34 to 35 in Crispy. However for Woof I just implemented it so 1 through 34 get coerced to 0 instead. I think I might like this better. Any opinions? I may go back and update Crispy if the latter approach is preferred.